src/master-playlist-controller.js

```diff
@@ -1725,11 +1728,23 @@ export class MasterPlaylistController extends videojs.EventTarget {
    * @return {Number} Desired buffer low water line in seconds
    */
   bufferLowWaterLine() {
-    const currentTime = this.tech_.currentTime();
+    const elapsedTime = this.elapsedSinceStart();
     const initial = Config.BUFFER_LOW_WATER_LINE;
     const rate = Config.BUFFER_LOW_WATER_LINE_RATE;
     const max = Math.max(initial, Config.MAX_BUFFER_LOW_WATER_LINE);
 
-    return Math.min(initial + currentTime * rate, max);
+    return Math.min(initial + elapsedTime * rate, max);
+  }
+
+  /**
+   * Calculates the time elapsed since last seek or initial start
+   *
+   * @return {Number} Elapsed time
+   */
+  elapsedSinceStart() {
```

Comment:
Minor, but since this is time elapsed since either start or last seek, maybe we should leave it somewhat generic and call it `elapsedTime()`?

Reply:
Sure, if you think that's descriptive enough, I'm fine with changing it.

Reply:
It probably is still a bit ambiguous, I may just be hooked onto it reading `Time` somewhere

Reply:
Yeah, it's hard to think of a good name that's not super long. I'd say maybe like a `timeSinceEmptyBuffer` but I'm not sure that accurately encapsulates the purpose of the function.

Reply:
Renamed to `elapsedTime`.